clang/runtime/dpct-rt/include/dpct/bindless_images.hpp

```diff
@@ -146,6 +146,218 @@ class image_mem_wrapper {
   image_mem_wrapper *_sub_wrappers{nullptr};
 };
 
+/// @brief  The wrapper structure for external memory handle desc
+struct external_mem_handle_desc {
+  int file_descriptor;
+  void *handle;
```

Comment:
```suggestion
#ifdef _WIN32
  void *handle = nullptr;
#else
  int handle = -1;
#endif
```

Reply:
**file_descriptor** member is only for Linux
**handle** member is only for Win32

So, **handle** will never be used on Linux

Reply:
On Win32, `handle` is a win32 handle with the type of `void *`.
On Linux, `handle` is a fd with the type of `int`.